src/ManageCourses.ApiClient/ManageCoursesApiClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net.Http.Headers;
+
+namespace GovUk.Education.ManageCourses.ApiClient
+{
+    public partial class ManageCoursesApiClient
+    {
+        partial void PrepareRequest(System.Net.Http.HttpClient client, System.Net.Http.HttpRequestMessage request, string url)
```

Comment:
url is not used

Reply:
Swagger insists on the method signature I'm afraid :)